src/server/common/forum_user_handler.cpp

```diff
@@ -207,6 +207,15 @@ std::string fuh::get_tournaments(){
 	return conn_.get_tournaments();
 }
 
+void fuh::get_game_history(boost::asio::io_service& io_service, server_base& s_base, socket_ptr player_socket, int player_id, int offset) {
+	boost::asio::post([this, &s_base, player_socket, player_id, offset, &io_service] { 
+		auto doc = conn_.get_game_history(player_id, offset);
+		boost::asio::post(io_service, [player_socket, &s_base, doc = std::move(doc)]{
```

Comment:
No need to move this into the lambda. Can just do:
```cpp
boost::asio::post(io_service, [player_socket, &s_base, doc = conn_.get_game_history(player_id, offset)] {
```

Reply:
Updated as well.